corehq/apps/sms/tasks.py

```diff
@@ -162,8 +162,10 @@ def handle_domain_specific_delays(msg, domain_object, utcnow):
 
 
 def message_is_stale(msg, utcnow):
-    oldest_allowable_datetime = \
-        utcnow - timedelta(hours=settings.SMS_QUEUE_STALE_MESSAGE_DURATION)
+    sms_stale_duration = msg.custom_metadata.get('sms_stale_after', None) if msg.custom_metadata else None
+    sms_stale_duration = sms_stale_duration \
+        if sms_stale_duration is not None else settings.SMS_QUEUE_STALE_MESSAGE_DURATION
```

Comment:
This constant already existed, but would you consider renaming it something like `SMS_QUEUE_DAYS_UNTIL_STALE`? Especially now that this and `MIN_SMS_STALE_INTERVAL` use different units - accidentally double-converting a value between hours and days is an easy mistake to make.

Reply:
I read this PR out of order, looks like this and `MIN_SMS_STALE_INTERVAL` are now all in hours, but `SMS_QUEUE_HOURS_UNTIL_STALE` would still be more clear.